pkg/networkmanager/interfaces.js

```diff
@@ -1722,6 +1722,19 @@ PageNetworking.prototype = {
             self.rx_plot.resize();
             self.tx_plot.resize();
         });
+
+        const desc = _("Network bonding is a method to combine or aggregate network interfaces to provide a logical interface with higher throughput or redundancy.");
+        const lm = _("Learn more");
+        const url = "https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_systems_using_the_rhel_8_web_console/managing-networking-in-the-web-console_system-management-using-the-rhel-8-web-console#bond-modes_configuring-network-bonds-using-the-web-console";
```

Comment:
I wonder if this should go to 7.1, rather than 7.1.2. So _configuring-network-bonds_ rather than _bond-modes_
Even though the bond modes are probably the hardest part, I think it's good to link to the higher level bond topic that explains the entire concept.

Reply:
+1 from me